pheweb/serve/react/js/tables.js

```diff
@@ -10,39 +10,106 @@ const variantSorter = (a, b) => {
     return v1[1] > v2[1] ? 1 : -1
 }
 
+const naSorter = (a, b) => {
```

Comment:
Is this a "react way" of doing things. Seems these definitions are used specifically only in their own components. Wouldn't it be easier to read and modify if these were in the component definition as they are not shared?

Reply:
These are indeed not shared and just used in the tables. Because the table code will be massive I put it separate from the component

Reply:
And the table code is actually shared with components